gtfs/spec/en/reference.md

```diff
@@ -342,41 +342,41 @@ When calculating an itinerary, GTFS-consuming applications interpolate transfers
 
 |  Field Name | Type | Presence | Description |
 |  ------ | ------ | ------ | ------ |
-|  `pathway_id` | ID | **Required** | The `pathway_id` field contains an ID that uniquely identifies the pathway. The `pathway_id` is used by systems as an internal identifier of this record (e.g., primary key in database), and therefore the `pathway_id` must be dataset unique. <br> Different pathways can go from the same `from_stop_id` to the same `to_stop_id`. For example, this happens when two escalators are side by side in opposite direction, or when a stair is nearby and elevator and both go from the same place to the same place.|
-|  `from_stop_id` | ID referencing `stops.stop_id` | **Required** | Location at which the pathway begins. It contains a `stop_id` that identifies a platform, entrance/exit, generic node or boarding area from the `stops.txt` file.|
-|  `to_stop_id` | ID referencing `stops.stop_id` | **Required** | Location at which the pathway ends. It contains a `stop_id` that identifies a platform, entrance/exit, generic node or boarding area from the `stops.txt` file.|
-|  `pathway_mode` | Enum | **Required** | Type of pathway between the specified (`from_stop_id`, `to_stop_id`) pair. Valid values for this field are: <br><br>`1` - Walkway. <br>`2` - Stairs. <br>`3` - Moving sidewalk/travelator. <br>`4` - Escalator. <br>`5` - Elevator. <br>`6` - Fare gate (or payment gate): A pathway that crosses into an area of the station where a proof of payment is required (usually via a physical payment gate). Fare gates may either separate paid areas of the station from unpaid ones, or separate different payment areas within the same station from each other. This information can be used to avoid routing passengers through stations using shortcuts that would require passengers to make unnecessary payments, like directing a passenger to walk through a subway platform to reach a busway. <br>`7`-  Exit gate: Indicates a pathway exiting an area where proof-of-payment is required into an area where proof-of-payment is no longer required.|
-|  `is_bidirectional` | Enum | **Required** | Indicates in which direction the pathway can be used:<br><br>`0` - Unidirectional pathway, it can only be used from `from_stop_id` to `to_stop_id`.<br>`1` - Bidirectional pathway, it can be used in the two directions.<br><br>Fare gates (`pathway_mode=6`) and exit gates (`pathway_mode=7`) cannot be bidirectional.|
+|  `pathway_id` | ID | **Required** | Identifies a pathway. Used by systems as an internal identifier for the record. Must be unique in the dataset. <br><br> Different pathways may have the same values for `from_stop_id` and `to_stop_id`.<hr>_Example: When two escalators are side-by-side in opposite directions, or when a stair set and elevator go from the same place to the same place, different `pathway_id` may have the same `from_stop_id` and `to_stop_id` values._|
```

Comment:
IMHO it is related to the use of `pathway_id` since it is describing the primary ID (or what is not the primary ID) of pathways.txt. The current description clarifies that `from_stop_id` and `to_stop_id` are not part of the primary ID, and thus can be the same for different `pathway_id`. If we had primary and foreign ID descriptions in GTFS as suggested in #266,  we could get rid of this description altogether and simply specify `pathway_id` as the primary ID. For now, I would favor keeping it in `pathway_id` also to avoid duplication of the description in both `from_stop_id` and `to_stop_id`.